components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3015,27 +3019,16 @@ void RewardsServiceImpl::ShowNotification(
     callback(ledger::type::Result::LEDGER_OK);
 }
 
+// OnlyAnonWallet is used to indicate that a particular region does not support
+// external wallets, and specifically it was used to modify the UI for users in
+// JP to show "BAP" instead of "BAT". When we are sure that those branches are
+// no longer needed, this function should be removed.
 bool RewardsServiceImpl::OnlyAnonWallet() const {
-  int32_t current_country = country_id_;
-  if (!current_country) {
-    current_country =
-        country_codes::GetCountryIDFromPrefs(profile_->GetPrefs());
-  }
-
-  for (const auto& country : kOnlyAnonWalletCountries) {
-    if (country.length() != 2) {
-      continue;
-    }
-
-    const int id = country_codes::CountryCharsToCountryID(
-        country.at(0), country.at(1));
-
-    if (id == current_country) {
-      return true;
-    }
-  }
-
+#if defined(OS_ANDROID)
+  return GetExternalWalletType() == ledger::constant::kWalletBitflyer;
```

Comment:
Will we still be showing BAP instead of BAT on android?

Reply:
For the first release, yes - Android will follow later.